brozzler/ydl.py

```diff
@@ -33,9 +33,11 @@
 
 thread_local = threading.local()
 
-YTDLP_PROXY = ""
-MAX_YTDLP_ATTEMPTS = 4
-YTDLP_WAIT = 10
+
+ytdlp_proxy = ""
+ytdlp_tmp = "/tmp"
+ytdlp_wait = 10
+max_ytdlp_attempts = 1
```

Comment:
Changing these to lowercase doesn't fix the problem. You're still redefining what should be a constant; it's just harder to tell that it should be a constant because you made it lowercase.

`max_ytdlp_attempts` is module-level scope. So when you redefine `max_ytdlp_attempts` from `1` to `4` inside of `should_ytdlp`, it does not revert back to `1` after exiting the function. It will remain as `4`, which breaks the entire point of what you are trying to do.